### ISSUE TITLE
nglview fails to recognize "phe" resname

### ISSUE THREAD

**Issue by hainm** [arose/ngl#137]
![phe](https://cloud.githubusercontent.com/assets/4451957/15728689/25b6f4de-282c-11e6-8a53-8e1127d05cbc.png)

this is stripped pdb file

```python
REMARK   1 CREATED WITH MDTraj 1.7.2, 2016-06-01
CRYST1   26.790   26.791   26.790  90.00  90.00  90.00 P 1           1 
MODEL        0
ATOM      1  C1  sol A   1       0.860  24.880  21.210  1.00  0.00          C   
ATOM      2  C2  sol A   1       0.530  23.520  21.220  1.00  0.00          C   
ATOM      3  C3  sol A   1       0.400  22.840  22.430  1.00  0.00          C   
ATOM      4  C4  sol A   1       0.590  23.520  23.630  1.00  0.00          C   
ATOM      5  C5  sol A   1       0.920  24.870  23.630  1.00  0.00          C   
ATOM      6  C6  sol A   1       1.060  25.550  22.420  1.00  0.00          C   
ATOM      7  O1  sol A   1       0.460  22.850  24.810  1.00  0.00          O   
ATOM      8  H1  sol A   1       0.970  25.410  20.270  1.00  0.00          H   
ATOM      9  H2  sol A   1       0.380  23.000  20.280  1.00  0.00          H   
ATOM     10  H3  sol A   1       0.140  21.790  22.420  1.00  0.00          H   
ATOM     11  H4  sol A   1       1.070  25.400  24.560  1.00  0.00          H   
ATOM     12  H5  sol A   1       1.310  26.610  22.410  1.00  0.00          H   
ATOM     13  H6  sol A   1       0.620  23.450  25.560  1.00  0.00          H   
ATOM     14  C1  phe A   2       4.730  11.870  23.600  1.00  0.00          C   
ATOM     15  C2  phe A   2       4.490  12.310  24.910  1.00  0.00          C   
ATOM     16  C3  phe A   2       3.190  12.620  25.320  1.00  0.00          C   
ATOM     17  C4  phe A   2       2.130  12.490  24.420  1.00  0.00          C   
ATOM     18  C5  phe A   2       2.370  12.050  23.120  1.00  0.00          C   
ATOM     19  C6  phe A   2       3.660  11.740  22.710  1.00  0.00          C   
ATOM     20  O1  phe A   2       0.870  12.790  24.820  1.00  0.00          O   
ATOM     21  H1  phe A   2       5.740  11.630  23.290  1.00  0.00          H   
ATOM     22  H2  phe A   2       5.320  12.410  25.600  1.00  0.00          H   
ATOM     23  H3  phe A   2       3.020  12.970  26.330  1.00  0.00          H   
ATOM     24  H4  phe A   2       1.540  11.940  22.420  1.00  0.00          H   
ATOM     25  H5  phe A   2       3.840  11.400  21.700  1.00  0.00          H   
ATOM     26  H6  phe A   2       0.240  12.650  24.090  1.00  0.00          H   
```

**Comment by arose** [comment]
Yeah, I assumed residue names in files are uppercase. Will fix.

**Comment by arose** [comment]
fixed 96aacdc300d399050b03431977e954d3b717eb99
